File: RepositoryPattern/SampleApplicationForGM/myfeature/src/main/java/com/example/myfeature/MyFeatureService.java
package com.example.myfeature;

import com.example.myfeature.models.DummyResponse;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface MyFeatureService {

    @GET("v1/employees")
    Observable<List<DummyResponse>> getDummyRestResponse();
}

File: RepositoryPattern/SampleApplicationForGM/myfeature/src/main/java/com/example/myfeature/MyFeatureService.kt
package com.example.myfeature

import com.example.myfeature.models.DummyResponse
import io.reactivex.Observable
import retrofit2.http.GET

interface MyFeatureService {
    @get:GET("v1/employees")
    val dummyRestResponse: Observable<List<DummyResponse>>
}